tests/lax_test.py

```diff
@@ -2348,6 +2348,16 @@ def testBroadcastInDim(self):
     make_const = lambda: lax.broadcast_in_dim(arr, (2, 1, 3), (0, 2))
     self._Check(make_const, expected)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_prim={}_indexdtype={}"
+       .format(prim, np.dtype(index_dtype).name),
+       "index_dtype": dtypes.canonicalize_dtype(index_dtype), "prim": prim}
+      for index_dtype in jtu.dtypes.all_inexact + jtu.dtypes.boolean
+      for prim in [lax.argmin_p, lax.argmax_p]))
+  def testArgMinMaxIndexDtypeError(self, prim, index_dtype):
+    with self.assertRaisesRegex(TypeError,
+                                "index_dtype must be an integer type"):
+      prim.bind(np.ones((2, 2)), axes=(0,), index_dtype=index_dtype)
```

Comment:
Wouldn't it be better to test `lax.argmin` instead, since it is the public API?

Reply:
You're right, this is better.